index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Pizza Calculator</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+
+
+<body>
+<h1>Cheese Pizza Calculator</h1>
+<div class="hero">
+    <p>"We love pizza. You love pizza. Don't skip it! But definitely be mindful of what's going into your body — that's true for all foods, pizza or otherwise! Here's what goes into some of your favorite slices: a small cheese pie varies from restaurant to restaurant, so we're giving you a good handful of options to look at from BJ's Brewhouse to California Pizza Kitchen to Domino's.
+
```

Comment:
Looks like you are trying to do a line break here? Since the computer won't read all this spacing, would recommend you separate the sentences into two different `p` elements or use a `<br/>` tag.